### PR TITLE
Add tags to ECR repos

### PR DESCRIPTION
**Pull request by kohidave**
Adding an app and project tag to ECR repos.

This will make it easier to do some permission checks (only be able to push to repos with the project tag).

### CODE REVIEW

File: templates/project/cf.yml [hunk 1]
```diff
@@ -86,15 +88,21 @@ Resources:
     Type: AWS::ECR::Repository
     Properties:
       RepositoryName: {{$project}}/{{$app}}
+      Tags:
+        -
+          Key: ecs-project
+          Value: {{$project}}
+        -
+          Key: ecs-app
```
Comment (efekarakus) [inline templates/project/cf.yml]: We use `ecs-application` in the LBFargateApp template:
https://github.com/aws/amazon-ecs-cli-v2/blob/c80b45a10e2e1d51cd38ed5252e06fd0abb7bdce/internal/pkg/deploy/cloudformation/stack/tags.go#L9

Could we use that instead?

File: internal/pkg/deploy/cloudformation/testdata/rendered_project_cfn_template.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+AWSTemplateFormatVersion: '2010-09-09'
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/testdata/rendered_project_cfn_template.yml]: nit: should this file be in the `testdata` directory under the `stack` pkg?

File: templates/project/cf.yml (hunk 1 above)
Comment (kohidave) [reply templates/project/cf.yml]: Ah - good point. Updated that.

File: internal/pkg/deploy/cloudformation/testdata/rendered_project_cfn_template.yml (hunk 2 above)
Comment (kohidave) [reply internal/pkg/deploy/cloudformation/testdata/rendered_project_cfn_template.yml]: sure

**Pull request merged**